certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Baron Blue",
+    "description": "Baron Blue allows you to perform complex operations in only one action. Convert JSON, XML, CSV, XLS, XLSX, and YAML between themselves in any combination. Convert to and from Excel Dates, find information in strings, find content-types and extensions, convert IP addresses (range, CIDR), and much more. New actions come monthly, and if you\u2019re a part of the manueltgomes.com/club, you\u2019ll get the connector for free, so be sure to check it out.",
+    "version": "1.0",
+    "contact": {
+      "name": "Manuel T Gomes Club Support",
+      "url": "https://manueltgomes.com/support",
+      "email": "help@manueltgomes.com"
+    }
+  },
+  "host": "api.manueltgomes.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.manueltgomes.com/baron-blue"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://manueltgomes.com/privacy-policy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/convert/string/content_type/to/extension/": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@mtgomes you can change `default` descriptions for response codes to `OK, Bad Request, etc` to make it more clear.

Reply:
@mtgomes Please change the description from "default" to "OK" or "Bad Request" .... etc